pom.xml

```diff
@@ -388,7 +387,7 @@
                 <executions>
                     <execution>
                         <goals>
-                            <goal>validate</goal>
+                            <goal>format</goal>
```

Comment:
Absolutely.  I cannot have my local machine failing to format with the plugin.  I'm not going to use an IDE to do that.  There is a complete difference between development and CI.  If you want validate in CI, we just have to configure the travis build to run the validate.  It's really easy to do.  If it helps to see how that might work, look over at mybatis and how I ran specialized builds in travis there doing many things using bash of all things as well.  All mybatis has this now.  Here is a link to the parent.

https://github.com/mybatis/parent

There generally has to be a middle ground here.  What I need to occur on my local is automation without blocking.  What we need on CI is blocking.  I agree there.  We don't want people submitting changes unformatted which would occur otherwise.  As to any concern if CI ever was doing formatting that pretty much doesn't matter at all and never really should.  It's the source in the REPO we care about.  CI builds are rather mute.  I have this formatter running in 100+ separate projects with a major bank and that is fully accepted.  What we have an issue with is clear, code gets to master not formatted and then is formatted after the fact so that is where CI comes into play in blocking merge to master.  Even here, it's just that we force the merge to master to do the same, block unformatted code.  Pretty simple.  The key is we have to remember we don't care what peoples local repos look like.  We only care at master.  And because we can enforce travis do it, users will know way before they attempt a pull as we can block the build.